package.json

```diff
@@ -32,12 +32,14 @@
     "url": "https://github.com/buildo/react-placeholder/issues"
   },
   "homepage": "https://github.com/buildo/react-placeholder",
-  "dependencies": {},
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "lodash.omit": "^4.5.0"
```

Comment:
```js
const { children, ready, firstLaunchOnly, type, ...props } = this.props;
```

actually this is not ok as eslint would complain... I'll keep `omit` for now (as we're doing for every other component for this same stupid reason...)